termwiz/src/surface/line/clusterline.rs

```diff
@@ -9,17 +9,17 @@ use std::num::NonZeroU8;
 
 #[cfg_attr(feature = "use_serde", derive(Serialize, Deserialize))]
 #[derive(Debug, Clone, PartialEq)]
-struct Cluster {
-    cell_width: u16,
-    attrs: CellAttributes,
+pub struct Cluster {
```

Comment:
hopefully the set of changes in this file are not needed with `lines_to_escapes`